Warnings with gcc 9.2 in Trilinos snapshot comparing span()

In the kokkos snapshot in trilinos, when building with with gcc 9.2, getting the following warnings:
```
/ascldap/users/rppawlo/Trilinos/packages/kokkos/core/src/Kokkos_CopyViews.hpp:1261:19: warning: comparison of integer expressions of different signedness: ‘size_t’ {aka ‘long unsigned int’} and ‘int’ [-Wsign-compare]
 1261 |     if(dst.span() < std::numeric_limits<int>::max()) {
/ascldap/users/rppawlo/Trilinos/packages/kokkos/core/src/Kokkos_CopyViews.hpp:1293:17: warning: comparison of integer expressions of different signedness: ‘size_t’ {aka ‘long unsigned int’} and ‘int’ [-Wsign-compare]
 1293 |   if(dst.span() > std::numeric_limits<int>::max()) {
```
The line numbers are different in current kokkos repo but it looks like the offending code is still present.

--- a/core/src/Kokkos_CopyViews.hpp
+++ b/core/src/Kokkos_CopyViews.hpp
@@ -1444,7 +1444,7 @@ inline void deep_copy(
         ViewTypeFlat;
 
     ViewTypeFlat dst_flat(dst.data(), dst.size());
-    if (dst.span() < std::numeric_limits<int>::max()) {
+    if (dst.span() < static_cast<size_t>(std::numeric_limits<int>::max())) {
       Kokkos::Impl::ViewFill<ViewTypeFlat, Kokkos::LayoutRight,
                              typename ViewType::execution_space,
                              ViewTypeFlat::Rank, int>(dst_flat, value);
@@ -1485,7 +1485,7 @@ inline void deep_copy(
   typedef typename std::conditional<
       ViewType::Rank == 0, typename ViewType::uniform_runtime_type,
       typename ViewType::uniform_runtime_nomemspace_type>::type ViewTypeUniform;
-  if (dst.span() > std::numeric_limits<int>::max()) {
+  if (dst.span() > static_cast<size_t>(std::numeric_limits<int>::max())) {
     if (iterate == Kokkos::Iterate::Right)
       Kokkos::Impl::ViewFill<ViewTypeUniform, Kokkos::LayoutRight,
                              typename ViewType::execution_space, ViewType::Rank,

